internal/jujuclient/cloud.go

```diff
@@ -103,26 +96,16 @@ func (c Connection) RevokeCredential(ctx context.Context, cred names.CloudCreden
 	out := jujuparams.ErrorResults{
 		Results: make([]jujuparams.ErrorResult, 1),
 	}
-	if c.SupportsCheckCredentialModels() {
-		in := jujuparams.RevokeCredentialArgs{
-			Credentials: []jujuparams.RevokeCredentialArg{{
-				Tag:   cred.String(),
-				Force: true,
-			}},
-		}
-		if err := c.CallHighestFacadeVersion(ctx, "Cloud", []int{7, 3}, "", "RevokeCredentialsCheckModels", &in, &out); err != nil {
-			return errors.E(op, jujuerrors.Cause(err))
-		}
-	} else {
-		in := jujuparams.Entities{
-			Entities: []jujuparams.Entity{{
-				Tag: cred.String(),
-			}},
-		}
-		if err := c.Call(ctx, "Cloud", 1, "", "RevokeCredentials", &in, &out); err != nil {
-			return errors.E(op, jujuerrors.Cause(err))
-		}
+	in := jujuparams.RevokeCredentialArgs{
+		Credentials: []jujuparams.RevokeCredentialArg{{
+			Tag:   cred.String(),
+			Force: true,
+		}},
 	}
+	if err := c.CallHighestFacadeVersion(ctx, "Cloud", []int{7}, "", "RevokeCredentialsCheckModels", &in, &out); err != nil {
```

Comment:
I assume this can be simplified because JIMM only supports recent versions of Juju controller so they will definitely have this method?